platforms/documentation/docs/src/docs/userguide/redirects/tutorial_gradle_command_line.adoc

```diff
@@ -0,0 +1,3 @@
+++++
+<meta http-equiv="refresh" content="0;URL=command_line_interface_basics.html">
```

Comment:
💅 This redirect differs from the permanent redirect that CF had, don't be surprised if visiting `/current/userguide/tutorial_gradle_command_line.html` sends you to `/current/userguide/command_line_interface.html` instead of `/current/userguide/command_line_interface_basics.html`

Not saying this should stay the same, just pointing out that testing this may lead to confusion...

Reply:
This redirect is correct as it takes you to the tutorial portion of the new doc.